client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -241,7 +241,8 @@ const Checkout = React.createClass( {
 				: '/checkout/thank-you/plans';
 		} else if ( selectedSite.options.is_domain_only && cartItems.hasDomainRegistration( cart ) && ! cartItems.hasPlan( cart ) ) {
 			const domainItem = find( cartItems.getAll( cart ), isDomainRegistration );
-			return domainManagementEdit( selectedSite.slug, domainItem.meta );
+
+			return domainManagementList( domainItem.meta );
```

Comment:
I was juggling between `selectedSite.slug` and `domainItem.meta`. Finally I decided to pick `domainItem.meta` hoping that primary domain change lands soon :)

I can update to use `selectedSite.slug` and add todo comment to make it work for the time being.